eng/tox/tox.ini

```diff
@@ -281,7 +281,7 @@ commands =
 
 
 [testenv:sphinx]
-description=Builds a package's documentation with sphinx
+description="Builds a package's documentation with sphinx"
```

Comment:
My editor (which probably needs update but lets leave that aside) starts to run the formatting together as string due to the `'` that is present in these unquoted lines. Adding quotes doesn't change anything but enables my editor to render syntax highlighting for the rest of the file, so I figured I'd just patch this up real quick.